### PR TITLE
Mh/import superhero

### PR DESCRIPTION
**Pull request by meenaHaque**
**Homework - 23/10/20**

- Create superheroes file and import this into app.js 
- Refactor map function to iterate over superhero array in main app
- Add inline style in JSX

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,36 +1,13 @@
...
-// function App() {
-//   return (
-//     <div className="App">
-//       {superHeros.map(hero => {
-//          return <h1>{hero.name}</h1>
-//       })}
-//     </div>
-//   );
-// }
+import Superheros from './Superheros'
```
Comment (VladFiliucov) [inline src/App.js]: Usually we capitalize component names. Collections should start with lowercase letter. So here we should have `superheroes` instead. Components starting with capital letter is a convention we use, so if I was working on this file and Superheroes (with capital S) where coming from a package or library I would try to render them like this `<Superheroes />`

**Review by VladFiliucov — CHANGES_REQUESTED** [review]
Almost there. To understand better the naming conventions I'm going to create an issue in this repo with optional homework. This it totally optional so if you have capacity to do it - great, otherwise we can pair on in our next session. https://github.com/meenaHaque/frontend-superheros/issues/3

File: src/App.js (hunk 1 above)
Comment (meenaHaque) [reply src/App.js]: Have amended this to the correct casing and have added some inline style to the JSX :)

**Review by VladFiliucov — APPROVED** [review]
🚢  it!

**Pull request merged**